### PR TITLE
Data visualization of a scientific object for a selected variable

### PR DESCRIPTION
**Pull request by Gouga34**

### CODE REVIEW

File: views/scientific-object/data_visualization.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            ?>
+            </div>
+        <br/>
+        
+            <div class="row">
+                <div class="col-md-6">
+                    <?=
+                    \kartik\date\DatePicker::widget([
+                        'name' => 'dateStart',
+                        'options' => ['placeholder' => 'Enter date start'],
```
Comment (annetireau) [inline views/scientific-object/data_visualization.php]: could you add translation.

File: views/scientific-object/data_visualization.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                        ]
+                    ])
+                    ?>
+                </div>
+                <div class="col-md-6">
+                    <?=
+                    \kartik\date\DatePicker::widget([
+                        'name' => 'dateEnd',
+                        'value' => isset($dateEnd) ? $dateEnd : null,
+                        'options' => ['placeholder' => 'Enter date end'],
```
Comment (annetireau) [inline views/scientific-object/data_visualization.php]: could you add translation.

Comment (Gouga34) [reply views/scientific-object/data_visualization.php]: done

File: views/scientific-object/data_visualization.php (hunk 1 above)
Comment (Gouga34) [reply views/scientific-object/data_visualization.php]: done

**Pull request merged**